atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -138,14 +138,19 @@ def generate_auto_labels(self, auto_labels):
                 generated[old] = generated[new]
 
         for lbl in auto_labels:
-            if lbl in self.labels:
-                self.log.info("label %r is set explicitly, not using generated value", lbl)
-                continue
-
-            if lbl in generated:
-                self.labels[lbl] = generated[lbl]
-            else:
-                self.log.warning("requested automatic label %r is not available", lbl)
+            for _, llist in Labels.LABEL_NAMES.items():
+                if lbl in llist:
```

Comment:
`llist` here is a tuple of old-style and new-style labels from `LABEL_NAMES`. An autolabel would surely be in the tuple